src/connectors/h1_collect.py

```diff
@@ -161,14 +170,12 @@ def insert_reports(landing_table, reports, recorded_at, dryrun):
                 'description': get_path(report, 'relationships.weakness.data.attributes.description'),
                 'external_id': get_path(report, 'relationships.weakness.data.attributes.external_id'),
                 'asset_identifier': get_path(report, 'relationships.structured_scope.data.attributes.asset_identifier'),
-                'awarded_amount': sum(
```

Comment:
this change looks a bit like a red herring — is this what fixed the bug, or the change below?

Reply:
This fixed the issue of awarded_amount and awarded_bonus_amount showing up for only the last page.

Reply:
why do you think so? do you see a logic difference between the two versions I am missing?